sdk/core/core-client/test/deserializationPolicy.spec.ts

```diff
@@ -68,21 +69,20 @@ describe("deserializationPolicy", function() {
       assert.isUndefined(response.parsedHeaders);
     });
 
-    it.skip(`with xml response body, application/xml content-type, but no operation spec`, async function() {
+    it(`with xml response body, application/xml content-type, but no operation spec`, async function() {
       const response = await getDeserializedResponse({
         headers: { "content-type": "application/xml" },
         bodyAsText: `<fruit><apples>3</apples></fruit>`
       });
       assert.exists(response);
       assert.isUndefined(response.readableStreamBody);
       assert.isUndefined(response.blobBody);
-      assert.isUndefined(response.parsedBody);
```

Comment:
I am curious what leads to this change?

Reply:
When I converted the tests I rewrote some of the assertions... and because many test assertions are similar I was copy pasting a *lot*. Since the tests were disabled though I didn't notice my copypaste errors. 😅